buildSrc/build.gradle

```diff
@@ -47,6 +47,8 @@ if (project == rootProject) {
 Properties props = VersionPropertiesLoader.loadBuildSrcVersion(project.file('version.properties'))
 version = props.getProperty("elasticsearch")
 processResources {
+  inputs.file(project.file('version.properties'))
+  inputs.property("dynamic_elasticsearch_version", props.getProperty("elasticsearch"))
```

Comment:
Do we need this one? The ES version already comes from the file, right?

Reply:
Yes, we add snapshot and qualifier from parameters to it.